### PR TITLE
feat: add mew wallet

### PR DESCRIPTION
**Pull request by jmjles**

### CODE REVIEW

**Comment by jmjles** [comment]
@markdalgleish @nickbytes 
This is the new pr for mew wallet. I also made another pr for Enkrypt.

**Comment by DanielSinclair** [comment]
@jmjles Looks like we're missing `downloadUrls.qrCode`. Most teams point to a redirect catch-all or their landing page download.
Screenshot 2023-01-27 at 8 37 19 AM

File: packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}: MewWalletOptions): Wallet => ({
+  id: 'mewWallet',
+  name: 'MEW wallet',
+  iconUrl: async () => (await import('./mewWallet.svg')).default,
+  iconBackground: '#fff',
+  downloadUrls: {
+    android:
+      'https://play.google.com/store/apps/details?id=com.myetherwallet.mewwallet&referrer=utm_source%3Drainbow',
+    ios: 'https://apps.apple.com/app/apple-store/id1464614025?pt=118781877&mt=8&ct=rainbow',
+  },
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts:24]: Need `qrCode` here

File: packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+export interface MewWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const mewWallet = ({
+  chains,
+  shimDisconnect,
+}: MewWalletOptions): Wallet => ({
+  id: 'mewWallet',
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts:16]: For consistency, let's use `mew` as the `id` here

File: packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  name: 'MEW wallet',
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts:17]: Can we replace this with `MEW Wallet` for consistency, or is this the correct brand representation?

**Review by DanielSinclair — CHANGES_REQUESTED** [review]

**Comment by DanielSinclair** [comment]
I seem to be unable to connect to WalletConnect via the MEW Wallet app. Is there an alternative path?
IMG_5768

**Comment by gamalielhere** [comment]
@DanielSinclair wallet connect support will be coming this month.

**Comment by gamalielhere** [comment]
@DanielSinclair it shouldn't need wallet connect actually. You can test this by running the demo locally, accessing MEW wallet, going to the "Browse" feature and accessing the demo via the local network port.

**Comment by DanielSinclair** [comment]
@gamalielhere CI is failing. Will need you to run a `pnpm lint:fix`

**Comment by gamalielhere** [comment]
@DanielSinclair pushed those changes

**Review by DanielSinclair — APPROVED** [review]
Working as expected now 👍

**Pull request closed without merging**